README.md

```diff
@@ -1,24 +1,15 @@
 Chitter Challenge
 =================
 
```

Comment:
Nice readme, as you said maybe add some more instructions for those not familiar with cloning and running projects from GitHub. Otherwise pretty comprehensive 💯